model/model_test.go

```diff
@@ -80,6 +80,21 @@ func (*testModelSuite) TestModelBasic(c *C) {
 		PKIsHandle:  true,
 	}
 
+	view := &TableInfo{
+		ID:          1,
+		Name:        NewCIStr("v"),
+		Charset:     "utf8",
+		Collate:     "utf8_bin",
+		Columns:     []*ColumnInfo{column},
+		Indices:     nil,
+		ForeignKeys: nil,
+		PKIsHandle:  true,
+		View: &ViewInfo{
+			SelectStmt:      "SELECT 1 AS 'C' FROM 'DUAL'",
```

Comment:
😕 `SELECT 1 AS 'C' FROM 'DUAL'` is invalid whether ANSI_QUOTES is enabled or not. Do you mean to use `` ` `` instead of `'`?

| Quotation mark | With ANSI_QUOTES | Without ANSI_QUOTES |
|:-:|:-:|:-:|
| `'…'` | String literal | String literal |
| `"…"` | Identifier | String literal |
| `` `…` `` | Identifier | Identifier |

Reply:
my mistake。。。